src/current/v24.3/triggers.md

```diff
@@ -0,0 +1,505 @@
+	- The [`OLD`](#trigger-variables) table row affected by the SQL operation that activated the trigger. This variable applies only to `UPDATE` and `DELETE` triggers.
+	- `NULL`, which stops the SQL operation that activated the `BEFORE` trigger.
+- The function for an [`AFTER`](#trigger-conditions) trigger should return `NULL`, because its return value will be ignored.
+- The function must be defined before creating the trigger.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE OR REPLACE FUNCTION function_name()
+  RETURNS TRIGGER AS $$
+  BEGIN
+	...
+  END
+  $$ LANGUAGE PLpgSQL;
+~~~
+
+Refer to [Examples](#examples).
+
+#### Trigger variables
+
+The following trigger variables are automatically created for trigger functions, and can be used in the function body.
+
+|      Variable     |    Type    |                                                                                      Description                                                                                      |
+|-------------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `NEW`             | `RECORD`   | New table row resulting from the SQL operation. For `INSERT` triggers, this is the row that will be inserted. For `UPDATE` triggers, this is the row containing the updated values.   |
+| `OLD`             | `RECORD`   | Old table row affected by `UPDATE` and `DELETE` operations. For `UPDATE` triggers, this is the row that will be updated. For `DELETE` triggers, this is the row that will be deleted. |
+| `TG_NAME`         | `NAME`     | Name of the trigger that was activated.                                                                                                                                               |
+| `TG_WHEN`         | `STRING`   | When the trigger is set to activate: [`BEFORE` or `AFTER`](#trigger-conditions).                                                                                                      |
+| `TG_LEVEL`        | `STRING`   | Scope of trigger behavior: [`ROW`](#trigger-conditions).                                                                                                                              |
+| `TG_OP`           | `STRING`   | SQL operation that activated the trigger: [`INSERT`, `UPDATE`, or `DELETE`](#trigger-conditions).                                                                                     |
+| `TG_RELID`        | `OID`      | [`OID`]({% link {{ page.version.version }}/oid.md %}) of the table associated with the trigger.                                                                                       |
```

Comment:
There's also a [TG_RELNAME](https://www.postgresql.org/docs/current/plpgsql-trigger.html#PLPGSQL-DML-TRIGGER-TG-RELNAME) that is the same as `TG_TABLE_NAME`, and is deprecated in Postgres.

Reply:
If deprecated, is it ok to leave it out?

Reply:
Yeah I think that's probably fine.